app/components/post_list/post/body/content/index.tsx

```diff
@@ -105,6 +107,13 @@ const Content = ({isReplyPost, layoutWidth, location, post, theme}: ContentProps
                 );
             }
             break;
+        case contentType.permalink:
+            return (
```

Comment:
you should probably check if permalink previews are enabled before returning the component, otherwise continue returning null